dj_cqrs/admin.py

```diff
@@ -0,0 +1,44 @@
+class CQRSAdminMasterSyncMixin:
+    """
+    Mixin that includes, on AdminModel, a custom action that allows syncronization of master's
```

Comment:
Mixin that includes a custom action in AdminModel. This action allows synchronizing master's model items from Django Admin page,